### DISCUSSION TITLE
How to present zero values in VictoryPie chart?

### DISCUSSION THREAD

**Discussion by sanieduran** [FormidableLabs/victory#2405] [Q&A]
I would like to present data which contains 0 values using pie chart. When all the values are null, pie does not appear. I am trying to show the pie, but it is visible only for values greater than zero. How can I make it visible even if the percentage is null for all the values?

**Comment by gksander** [comment]
Interesting. What is a pie chart supposed to look like when the "aggregated" value is 0? I just checked a couple other charting libraries, and they also seem to make the chart "disappear" when all of the data is 0. Because the fundamental logic for pie charts is that each slice has an area of `(slice's y value) / (total of all y values)` and if `(total of all y values) === 0` we run into division-by-zero problems.

One option is to use `Number.EPSILON` instead of 0, something like `y => y || Number.EPSILON` so that there's always _something_ to divide by. In the case of "all values are 0", this would result in `N` equally-sized wedges.
